docs/spec/staking/hooks.md

```diff
@@ -3,17 +3,17 @@
 The staking module allow for the following hooks to be registered with staking events:
 
 ``` golang
-// event hooks for staking validator object
 type StakingHooks interface {
```

Comment:
This has the same just psuedo code problem as the old documentation has. Maybe you can describe what the hooks do, and when/where they are called? Also do we need a spec for the hooks system?

Reply:
Agreed, let's ditch the pseudocode in favor of just a list. I think it would be worth explaining the difference between `After` hooks & `Before` hooks.

Reply:
added a description and bulleted list 

```
Other modules may register operations to execute when a certain event has
occurred within staking.  These events can be registered to execute either
right `Before` or `After` the staking event (as per the hook name). The
following hooks can registered with staking: 
```